src/state/models/feeds/notifications.ts

```diff
@@ -523,9 +526,21 @@ export class NotificationsFeedModel {
   // =
 
   async _replaceAll(res: ListNotifications.Response) {
-    if (res.data.notifications[0]) {
-      this.mostRecentNotificationUri = res.data.notifications[0].uri
+    const latest = res.data.notifications[0]
+
+    if (latest) {
+      // not in action
+      this.mostRecentNotificationUri = latest.uri
+
+      const now = new Date()
+      const lastIndexed = new Date(latest.indexedAt)
+      const nowOrLastIndexed = now > lastIndexed ? now : lastIndexed
+
+      runInAction(() => {
```

Comment:
This `runInAction()` closure shouldnt be necessary (there hasn't been an await prior, at least that I can see in the diff)

Reply:
Ok I follow, fixing rn